tasks/libs/common/utils.py

```diff
@@ -485,7 +485,7 @@ def get_matching_pattern(ctx, major_version, release=False):
     pattern = rf"{major_version}\.*"
     if release or is_allowed_repo_nightly_branch(os.getenv("BUCKET_BRANCH")):
         pattern = ctx.run(
-            rf"git tag --list | grep -E '^{major_version}\.[0-9]+\.[0-9]+(-rc.*|-devel.*)?$' | sort -rV | head -1",
+            rf"git tag --list --merged {get_git_branch_name()} | grep -E '^{major_version}\.[0-9]+\.[0-9]+(-rc.*|-devel.*)?$' | sort -rV | head -1",
```

Comment:
I'd rather use [get_current_branch](https://github.com/DataDog/datadog-agent/blob/main/tasks/libs/common/git.py#L30) since I'll remove `get_git_branch_name` soon

Reply:
done thanks